Cargo.toml

```
@@ -27,6 +27,7 @@ criterion = "0.5"
 [features]
 laz = ["dep:laz"]
 laz-parallel = ["dep:laz", "laz/parallel"]
+copc = ["laz"]
```

Since there's no extra dependencies, I don't think we need to add a new feature. We can just include the `copc` module if `laz` is enabled.

```suggestion
laz = ["dep:laz"]
laz-parallel = ["dep:laz", "laz/parallel"]
```